assets/scss/_mixins.scss

```diff
@@ -11,3 +11,17 @@
   border-left: $border-width solid $error-border-color;
   padding: 0.75em 1em;
 }
+
+@mixin page-slice () {
+  @extend %contain-floats;
+  display: block;
+  max-width: 960px;
+  margin: 0 auto;
+  padding: 0 15px;
+  line-height: 1.5;
+  @include core-19();
```

Comment:
```
line-height: 1.5;
@include core-19();
```
Are added here as they are used to override `govuk_elements` selector - will add issue to remove once `govuk_elements` has been removed